src/languages/es.ts

```diff
@@ -784,6 +784,16 @@ export default {
             setDefaultFailure: 'No se ha podido configurar el método de pago.',
         },
         addBankAccountFailure: 'Ocurrió un error inesperado al intentar añadir la cuenta bancaria. Inténtalo de nuevo.',
+        getPaidFaster: 'Recibe pagos más rápido',
+        addPaymentMethod: 'Agregue un método de pago para enviar y recibir pagos directamente en la aplicación.',
+        getPaidBackFaster: 'Recibe tu dinero más rápido',
+        secureAccessToYourMoney: 'Acceso seguro a su dinero',
+        receiveMoney: 'Recibe dinero en tu moneda local',
+        expensifyWallet: 'Expensify Wallet',
+        sendAndReceiveMoney: 'Envía y recibe dinero desde tu Expensify Wallet.',
+        bankAccounts: 'Cuentas bancarias',
+        addBankAccountToSendAndReceive: 'Agregue una cuenta bancaria para enviar y recibir pagos directamente en la aplicación.',
+        addBankAccount: 'Agregar cuenta bancaria',
```

Comment:
Expensify Wallet should be `Billetera Expensify`.

The rest look alright if you'd like to check against these:

```
Cobra más rápido
Añade un método de pago para enviar y recibir pagos directamente en la aplicación.
Recibe tus pagos más rápido
Acceso seguro a tu dinero
Recibe dinero en tu moneda local
Billetera Expensify
Envía y recibe dinero desde tu Billetera Expensify.
Cuentas bancarias
Añade una cuenta bancaria para enviar y recibir pagos directamente en la aplicación.
Añadir cuenta bancaria
```